strr-examiner-web/app/stores/examiner.ts

```diff
@@ -99,6 +100,22 @@ export const useExaminerStore = defineStore('strr/examiner-store', () => {
     })
   }
 
+  const downloadReceipt = async (applicationNumber: string) => {
```

Comment:
Have you seen `downloadApplicationReceipt` in base layer composable? Maybe that could be used here?

Reply:
I have to hide the button in Payment Due status, I will add the condition.

Reply:
The function name got me thinking they want to download it. Thanks for sharing the req-s.

Reply:
I will do the cleanup and rename it.

Reply:
Done